### PR TITLE
[IMP] From travis to Github actions

### PR DESCRIPTION
**Pull request by etobella**
This is a test

### CODE REVIEW

**Comment by etobella** [comment]
@simahawk There is an issue about development status of edi_oca and edi_account. Should we change the development status of edi_oca to Beta?

**Comment by etobella** [comment]
@alexis-via runboat is not happy with the incompatibility between account_invoice_simple_pdf and account_invoice_import_invoice2data. I will remove simple_pdf from runboat. Sorry :wink:

**Comment by etobella** [comment]
Just one question @alexis-via Why they must be incompatible?

File: edi_backend_partner_oca/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "license": "LGPL-3",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "maintainers": ["LoisRForgeFlow"],
-    "development_status": "Beta",
+    "development_status": "Alpha",
```
Comment (simahawk) [inline edi_backend_partner_oca/__manifest__.py]: yeah, we could probably switch the base module to beta (pls do it in its own commit)

**Review by simahawk — COMMENTED** [review]
cooool!

**Review by simahawk — APPROVED** [review]
:heart:

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
What are we waiting?
